### PR TITLE
User factory girl for test fixtures

### PR DESCRIPTION
**Pull request by WillAbides**
This gets rid of the yml fixtures and replaces them with factory girl fixtures

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ class UsersController < ApplicationController
   def edit; end
 
   def update
-    if current_user.update_attributes(update_user_params)
+    params = update_user_params
+    user = current_user
```
Comment (andrew) [inline app/controllers/users_controller.rb]: that assignment seems unnecessary

**Comment by andrew** [comment]
👍

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (WillAbides) [inline app/controllers/users_controller.rb]: Thanks.  It's totally unnecessary.  I split it out to debug something and accidentally left it in.

**Comment by WillAbides** [comment]
I messed this up by having a local .env file with `PERSONAL_ACCESS_TOKENS_ENABLED=true` and `MINIMUM_REFRESH_INTERVAL=2` so a lot of tests are failing on ci that don't fail locally.  I'll push a fix later today.

**Comment by WillAbides** [comment]
It's fixed and ready to go now

**Comment by andrew** [comment]
👏

**Pull request merged**